### PR TITLE
add memcached wrapper docs

### PR DESCRIPTION
**Pull request by vetinari**
also: fix typo in redis docs

### CODE REVIEW

**Comment by vetinari** [comment]
:+1:

File: docs/user/check-ref/memcached_wrapper.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
+Redis
```
Comment (mohabusama) [inline docs/user/check-ref/memcached_wrapper.rst]: s/Redis/Memcached

File: docs/user/check-ref/memcached_wrapper.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+-----
+
+Read-only access to memcached servers is provided by the :py:func:`memcached` function.
+
+
+.. py:function:: redis([port=6379], [db=0])
```
Comment (mohabusama) [inline docs/user/check-ref/memcached_wrapper.rst]: memcached

**Comment by vetinari** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Pull request merged**